pkg/storaged/clevis-luks-passphrase.sh

```diff
@@ -7,18 +7,32 @@ fi
 
 DEV="$1"
 
-# The UUID that clevis uses for its luksmeta slots,
```

Comment:
Probably a follow-up, but such scripts should really be `set -e`, and preferrably `set -u` too. This points out much better when e. g. the `jose` program is not available, or some other assumption fails.

Reply:
Very good point, fixed.